src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -832,13 +868,13 @@ private void InheritNullableStateOfMember(int targetContainerSlot, int valueCont
                     int valueMemberSlot = -1;
                     if (valueContainerSlot > 0)
                     {
-                        int slot = GetOrCreateSlot(member, valueContainerSlot);
+                        int slot = GetOrCreateSlot(member, valueContainerSlot, createIfMissing: false);
```

Comment:
>createIfMissing: false [](start = 79, length = 22)

I am not sure if passing this value is going to have any useful effect. If we are not interested in allocating new slots, perhaps we should use ```VariableSlot``` API instead.